python/python_direct/ops.cpp

```diff
@@ -2399,6 +2416,36 @@ TensorView
     A TensorView of same dtype as `arg` and of shape `(C_i..., J, C_k...)` where
     the element at position `(i...,j,k...)` is equal to
     `arg[i,...,index[i,...,j,k,...],k,...]`.
+)",
+      py::return_value_policy::reference);
+  ops.def(
+      "pad",
+      pad_fn<py::list>,
+      py::arg("arg"),
+      py::arg("pad_widths"),
+      py::arg("value") = py::none(),
+      py::return_value_policy::reference);
+  ops.def(
+      "pad",
+      pad_fn<py::tuple>,
+      py::arg("arg"),
+      py::arg("pad_widths"),
+      py::arg("value") = py::none(),
+      R"(
+Pad a tensor.
+
+Parameters
+----------
+arg : TensorView
+pad_widths : list or tuple
+    The widths of the padding.
+value : Val, optional
+    The value to pad with. Default is None.
```

Comment:
nitpick: default arg for value is None, but the default padding value is `0` / `false`

Reply:
We have python default args be `None` that get translated into some default value like `0` or `false`.
I guess we should have the actual default value instead of the python default.